int64 sequence_id
string sequence_name
int64 class_id
string class_name

